src/plugins/vis_type_metric/public/metric_vis_renderer.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { MetricVisComponent } from './components/metric_vis_component';
+import { getI18n } from './services';
+import { VisualizationContainer } from '../../visualizations/public';
+
+export const metricVisRenderer = () => ({
```

Comment:
There's an `ExpressionRendererDefinition` type you can use here.

Reply:
actually, can't find it ...

Reply:
Oops sorry I had the name wrong; it's `ExpressionRenderDefinition`. not renderer

Reply:
here's the path to it `src/plugins/expressions/common/expression_renderers/types.ts`